wiki/en/Installation-for-Linux.md

```diff
@@ -54,22 +54,43 @@ For installers on other distributions, see their package managers and [Repology]
 
 ## Set up your hardware
 
+If no instance of a JACK server is running, one is normally started automatically.
+
+Jamulus will attach to a running JACK instance by name, using the `JACK_DEFAULT_SERVER` environment variable or `default` if the variable is not set.
+
+If your distribution uses JACK as the sound server, check out [how to configure JACK with QjackCtl](Installation-for-Linux#configure-jack-with-qjackctl).
+
+If your distribution uses PipeWire by default, check out the section on [configuring PipeWire](Installation-for-Linux#configure-pipewire).
+
 ### Configure JACK with QjackCtl
 
 Jamulus Clients need [JACK](https://jackaudio.org/) to run, but you need to configure that first. The recommended method is to use `QjackCtl`.
 
-1. Launch QjackCtl. You will see the **Qt JACK Control utility main page**
+1. Launch QjackCtl. You will see the **Qt JACK Control utility main page**.
 2. Configure your audio hardware as follows (the exact settings for JACK will depend on what your audio hardware is capable of):
 
-- Set the audio **Interface** to the one you want (there may be several in the list)
-- Set the **Sample Rate to 48000**
-- Set the **Frames/Period to 128** and Periods/Buffer at 2 at first
+- Set the audio **Interface** to the one you want (there may be several in the list).
+- Set the **Sample Rate to 48000**.
+- Set the **Frames/Period to 128** and **Periods/Buffer at 2** at first.
 
-Restart JACK for the new settings to take effect
+Restart JACK for the new settings to take effect.
+
+### Configure PipeWire
+
+The Jamulus Client requires a JACK sound server (native PipeWire support has not yet been implemented).
+PipeWire provides a compatibility layer that emulates the JACK server. When you launch the Jamulus Client, PîpeWire presents itself to Jamulus as a JACK server.
+You need to set your audio interface to the "pro-audio" profile and define PipeWire's "rate" and "quantum" parameters beforehand.
+To do so, follow these steps:
+1. Open `pavucontrol` (the audio mixer) and choose the "pro-audio" profile in the configuration tab.
+2. In a terminal use the two following commands:
+```bash
+pw-metadata -n settings 0 clock.force-rate 48000
+pw-metadata -n settings 0 clock.force-quantum 128
+```
 
 ### Start Jamulus
 
-With JACK running and configured, launch Jamulus.
+With JACK or PipeWire configured, launch Jamulus.
 
 If you get problems with sound breaking up (in particular XRUN errors reported by JACK/QjackCtl) try setting bigger values (e.g. 256 frames or 3 periods). Lower ones (e.g. 64 frames) could bring better performance but maybe more sound problems. See the [troubleshooting page](Client-Troubleshooting) otherwise.
```

Comment:
```suggestion
If you get problems with sound breaking up (in particular XRUN errors reported by JACK/QjackCtl) try setting bigger values (e.g. 256 frames or 3 periods). Lower ones (e.g. 64 frames) could bring better performance but might also bring more sound problems. See [the troubleshooting page](Client-Troubleshooting) otherwise.
```